[2024-09-02T07:08:34.115+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T07:08:34.137+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v5.second_task scheduled__2024-09-01T00:00:00+00:00 [queued]>
[2024-09-02T07:08:34.149+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v5.second_task scheduled__2024-09-01T00:00:00+00:00 [queued]>
[2024-09-02T07:08:34.149+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T07:08:34.179+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): second_task> on 2024-09-01 00:00:00+00:00
[2024-09-02T07:08:34.196+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2236) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T07:08:34.198+0000] {standard_task_runner.py:72} INFO - Started process 2249 to run task
[2024-09-02T07:08:34.196+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v5', 'second_task', 'scheduled__2024-09-01T00:00:00+00:00', '--job-id', '392', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpd_pf6e65']
[2024-09-02T07:08:34.200+0000] {standard_task_runner.py:105} INFO - Job 392: Subtask second_task
[2024-09-02T07:08:34.272+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v5.second_task scheduled__2024-09-01T00:00:00+00:00 [running]> on host efc48c282cbe
[2024-09-02T07:08:34.376+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='my_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-01T00:00:00+00:00'
[2024-09-02T07:08:34.377+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T07:08:34.397+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T07:08:34.398+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1']
[2024-09-02T07:08:34.409+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T07:08:34.410+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1
[2024-09-02T07:08:34.411+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T07:08:34.441+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T07:08:34.442+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v5, task_id=second_task, run_id=scheduled__2024-09-01T00:00:00+00:00, execution_date=20240901T000000, start_date=20240902T070834, end_date=20240902T070834
[2024-09-02T07:08:34.496+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T07:08:34.508+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
